src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -1519,6 +1541,13 @@ subroutine initialize_dyn_split_RK2(u, v, h, uh, vh, eta, Time, G, GV, US, param
       'Zonal Pressure Force Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
   CS%id_PFv = register_diag_field('ocean_model', 'PFv', diag%axesCvL, Time, &
       'Meridional Pressure Force Acceleration', 'm s-2', conversion=US%L_T2_to_m_s2)
+  CS%id_ueffA = register_diag_field('ocean_model', 'ueffA', diag%axesCuL, Time, &
+       'Effective U-Face Area', 'm^2', conversion = GV%H_to_MKS*US%L_to_m, &
```

Comment:
This is a very minor point, but I think that the conversion factor here should be `conversion=GV%H_to_m*US%L_to_m`, which would lead to an unscaled diagnostic in units of m^2 for both the Boussinesq and the non-Bousinesq cases. As written, this will resolve to m^2 for a Boussinesq case but kg m^-1 for the non-Boussinesq case.  If those really were the intended units of the diagnostic, the units argument would need to be changed to be consistent.  (See other examples in the code that deliberately use GV%H_to_MKS in their conversion factors for how to set their units strings, but I suspect that this is not what we want for this diagnostic.)

  A similar issue exists at lines 710 and  713 of MOM_dynamics_unsplit.F90 and lines 671 and 674 of MOM_dynamics_unsplit_RK2.F90.

  Apart from this one very minor set of corrections, this PR is now a thing of beauty, and it will be ready for acceptance after this last thing is resolved.  Well done, @sditkovsky!